mythril/analysis/solver.py

```diff
@@ -109,29 +108,49 @@ def get_transaction_sequence(global_state, constraints):
             minimize.append(transaction.call_data.calldatasize)
             minimize.append(transaction.call_value)
 
-            concrete_transactions[tx_id] = tx_template.copy()
-
         else:
             creation_tx_ids.append(tx_id)
 
     model = get_model(tx_constraints, minimize=minimize)
-
+    min_price_dict = {}  # type: Dict[str, int]
     for transaction in transactions:
         tx_id = str(transaction.id)
-
-        concrete_transactions[tx_id]["calldata"] = "0x" + "".join(
+        concrete_transaction = dict()  # type: Dict[str, str]
+        concrete_transaction["input"] = "0x" + "".join(
             [
                 hex(b)[2:] if len(hex(b)) % 2 == 0 else "0" + hex(b)[2:]
                 for b in transaction.call_data.concrete(model)
             ]
         )
-
-        concrete_transactions[tx_id]["call_value"] = (
-            "0x%x"
-            % model.eval(transaction.call_value.raw, model_completion=True).as_long()
-        )
-        concrete_transactions[tx_id]["caller"] = "0x" + (
+        value = model.eval(transaction.call_value.raw, model_completion=True).as_long()
+        concrete_transaction["value"] = "0x%x" % value
+        caller = "0x" + (
             "%x" % model.eval(transaction.caller.raw, model_completion=True).as_long()
         ).zfill(40)
 
-    return concrete_transactions
+        concrete_transaction["origin"] = caller
```

Comment:
The caller and origin are not the same

Reply:
We don’t have an origin, so I put in caller there